scripts/dotnet-cli-build/PrepareTargets.cs

```diff
@@ -166,25 +165,30 @@ public static BuildTargetResult CheckPackageCache(BuildTargetContext c)
 
             if (ciBuild)
```

Comment:
Does anyone set this anymore?  It used to be set in ci_build.cmd/sh. But those no longer exist.